modules/uptime-check/main.tf

```diff
@@ -73,8 +64,8 @@ resource "google_monitoring_alert_policy" "uptime_check_alert_policy" {
 metric.type="monitoring.googleapis.com/uptime_check/check_passed" AND metric.label.check_id="${google_monitoring_uptime_check_config.uptime_check_config[each.value.service_name].uptime_check_id}" AND resource.type="uptime_url"
       EOT
 
-      duration        = lookup(each.value, "uptime_duration", "300s")
-      threshold_value = lookup(each.value, "threshold", 1)
+      duration        = lookup(each.value, "alerts.uptime_duration", "300s")
```

Comment:
where is it getting `alerts` from in the `alerts.uptime_duration`?

Reply:
i've added an alets block to the actual uptime check, so we don't need two inputs `(uptime_checks and uptime_checks_alets)`. More logical to have that in the same object, imo :)

Reply:
Discussed in slack, need to be fixed